Sources/Apollo/RelayPaginationStrategy.swift

```diff
@@ -0,0 +1,124 @@
+#if !COCOAPODS
+import ApolloAPI
+#endif
+import Foundation
+
+/// A pagination strategy to be used with Relay-style cursor based pagination.
+public class RelayPaginationStrategy<
```

Comment:
Good call-out here, thanks for the mention.  I don't have a strong personal opinion here.  In Apollo Client a similar utility is called `relayStylePagination`.  So there's precedent in case that helps inform the decision.